float64 timeStamp    # ROS time-stamp
int64 encMotor       # Motor encoder
int64[] encFinger    # Finger encoders
